modules/distribution/resources/migration/wso2telco-hub-migration-client/src/main/resources/RemoveCustomHandler.sh

```diff
@@ -0,0 +1,27 @@
+echo "Please enter the path"
+read path
+lastChar=${path: -1}
+subPath='wso2telcohub-2.0.0/repository/deployment/server/synapse-configs/default/api/'
```

Comment:
remove the product directory from the subpath 
eg the there shoud be only internal paths like "repository/deployment/server/synapse-configs/default/api/'"